pom.xml

```diff
@@ -518,9 +518,9 @@
         <version.org.glassfish.jaxb.jaxb-xjc>${version.org.glassfish.jaxb}</version.org.glassfish.jaxb.jaxb-xjc>
         <version.org.glassfish.soteria>3.0.3</version.org.glassfish.soteria>
         <version.org.glassfish.web.jakarta.servlet.jsp.jstl>3.0.1-jbossorg-1</version.org.glassfish.web.jakarta.servlet.jsp.jstl>
-        <version.org.hibernate.commons.annotations>6.0.6.Final</version.org.hibernate.commons.annotations>
-        <version.org.hibernate>6.4.10.Final</version.org.hibernate>
-        <version.org.hibernate.search>7.1.2.Final</version.org.hibernate.search>
+        <version.org.hibernate.commons.annotations>7.0.0.Final</version.org.hibernate.commons.annotations>
+        <version.org.hibernate>6.6.1.Final</version.org.hibernate>
+        <version.org.hibernate.search>7.1.1.Final</version.org.hibernate.search>
```

Comment:
Hibernate Search should of been left as `7.1.2` but that isn't important since https://github.com/wildfly/wildfly/pull/18207 is also needed which upgrades search to `7.2.1`.  

CC @marko-bekhta